### DISCUSSION TITLE
Add more IGV tracks to the splice junctions browser?

### DISCUSSION THREAD

**Discussion by northwestwitch** [Clinical-Genomics/scout#5290] [Ideas]
Apart from the samples tracks, splice junction browser currently supports only gene tracks. One gene track under each sample track. I'm wondering if we should add more tracks, like custom tracks (local or remote). Perhaps just one copy on top of the view?

**Comment by dnil** [comment]
Did you get any feedback here? Convert to a poll perhaps?
